example/asm/shellcode.py

```diff
@@ -96,8 +96,13 @@
     patches = new_patches
 
 print patches
-for offset, raw in patches.items():
-    virt[offset] = raw
+if isinstance(virt, StrPatchwork):
```

Comment:
Yes, @fmonjalet  saw that as well, but I will not add these feature for the moment. I am 'testing' this new API for now